amgut/lib/data_access/test/test_ag_data_access.py

```diff
@@ -14,16 +19,114 @@ def test_authenticateWebAppUser(self):
         raise NotImplementedError()
 
     def test_addAGLogin(self):
-        raise NotImplementedError()
+        # test new user
```

Comment:
This is what `while exists is not None:` is for: It ensures that the email used is not already in there.